assets/blocks/restricted-content/enrolled/block.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "sensei-lms/enrolled-content",
+  "category": "sensei-lms",
+  "supports": {
+    "html": false,
```

Comment:
I think it's fine to allow html editing for these.

Reply:
We did some tests with it and there seems do be no useful use cases for this so we are going to keep it as it is.